src/lib/services/searchUtilActions/AasSearcher.ts

```diff
@@ -94,16 +95,19 @@ export class AasSearcher {
             return aasRegistryResult;
         }
 
-        const defaultResult = await this.multipleDataSource.getAasFromDefaultRepository(aasIdEncoded);
+        const defaultResult = await this.getAasFromDefaultRepository(aasIdEncoded);
         if (defaultResult) {
             return this.createAasResult(defaultResult);
         }
 
-        const potentiallyMultipleAas = await this.multipleDataSource.getAasFromAllRepos(aasIdEncoded);
-        if (potentiallyMultipleAas.length > 1) {
+        const potentiallyMultipleAas = await this.getAasFromAllRepositories(aasIdEncoded);
```

Comment:
For me it would be more readable this way:
```suggestion
        const potentiallyMultipleAas = await this.getAasFromAllRepositories(aasIdEncoded);
        if (potentiallyMultipleAas) {
            if (potentiallyMultipleAas.length > 1) {
                return this.createDiscoveryRedirectResult(searchInput);    
            } else {
                return this.createAasResult(potentiallyMultipleAas[0].aas);    
            } 
        } 
        return null;

Reply:
Added a slightly different version, is this okay as well?

Reply:
Yes thank you, makes it easier to read :)